src/Ztm.Zcoin.NBitcoin/Exodus/SimpleSendEncoder.cs

```diff
@@ -8,6 +8,26 @@ public sealed class SimpleSendEncoder : TransactionPayloadEncoder
     {
         public override int Type => SimpleSendV0.StaticId;
 
+        protected override void Encode(BinaryWriter writer, ExodusTransaction transaction)
+        {
+            if (!(transaction is SimpleSendV0))
+            {
+                throw new InvalidOperationException("The transaction type is not simple send v0.");
```

Comment:
Message is quite incorrect due to this class support any version of simple send.